server/service/src/log_service.rs

```diff
@@ -0,0 +1,85 @@
+use anyhow::Error;
+use repository::{KeyValueStoreRepository, KeyValueType, RepositoryError};
+use std::{fs, path::Path};
+
+use crate::{service_provider::ServiceContext, settings::Level};
+
+pub trait LogServiceTrait: Send + Sync {
+    fn get_log_file_names(&self) -> Result<Vec<String>, Error> {
+        let log_dir = Path::new("log");
```

Comment:
the log dir needs to come from settings, as this is configured in `local.yaml`.
Can we get the context here? Not sure how to fetch the settings!

Reply:
[0c8ac0b](https://github.com/openmsupply/open-msupply/pull/1897/commits/0c8ac0bba04685e49c48cb646fa29f4bab83254a)